lib/src/application/get_banner_bloc/banner_state.dart

```diff
@@ -8,10 +8,10 @@ part of 'banner_bloc.dart';
 @freezed
 abstract class BannerState with _$BannerState {
```

Comment:
Dont use abstract class for State or Events just use normal class

Reply:
finish